### PR TITLE
Add helpful links to base.html

### PR DESCRIPTION
**Pull request by sebbel**

### CODE REVIEW

**Comment by lpatmo** [comment]
@sebbel Thank you for starting the work adding some helpful info to `cbv3_django_prototype/cbv3_django_prototype/templates/base.html`! 

💯 to rebasing after #44 is merged 👍

**Comment by sebbel** [comment]
@lpatmo @billglover 
this can be reviewed, I have pulled master so that it's only a 1 file change

File: cbv3_django_prototype/cbv3_django_prototype/templates/base.html [hunk 1]
```diff
@@ -83,7 +83,22 @@
...
+      <h2>Application urls</h2>
+      <ul>
+        <li><a href="http://127.0.0.1:8000/api/v1/resources/">http://127.0.0.1:8000/api/v1/resources/ </a> - browse the API for resources</li>
+        <li><a href="http://127.0.0.1:8000/admin/">http://127.0.0.1:8000/admin/ </a> - login using superuser credentials</li>
+        <li><a href="http://127.0.0.1:8000/auth/obtain_token/">http://127.0.0.1:8000/auth/obtain_token/admin/</a> - get a token</li>
+        <li><a href="http://127.0.0.1:8000/auth/users/">http://127.0.0.1:8000/auth/users/</a> - create a user</li>
+      </ul>
+      <h2>docker-compose commands</h2>
+      <ul>
+        <li><code>docker-compose run --rm manage</code> createsuperuser - How to create a superuser</li>
```
Comment (sunu) [inline cbv3_django_prototype/cbv3_django_prototype/templates/base.html]: May be `createsuperuser` should be inside `<code/>` tags to be consistent with other lines.

File: cbv3_django_prototype/cbv3_django_prototype/templates/base.html [hunk 2]
```diff
@@ -83,7 +83,22 @@
...
       {% block content %}
-        <p>Use this document as a way to quick start any new project.</p>
+      <h1>Welcome to the codebuddies django-concept project</h1>
+      <p> Please find some useful commands below to help you get started on Django development</p>
+      <h2>Application urls</h2>
+      <ul>
+        <li><a href="http://127.0.0.1:8000/api/v1/resources/">http://127.0.0.1:8000/api/v1/resources/ </a> - browse the API for resources</li>
+        <li><a href="http://127.0.0.1:8000/admin/">http://127.0.0.1:8000/admin/ </a> - login using superuser credentials</li>
+        <li><a href="http://127.0.0.1:8000/auth/obtain_token/">http://127.0.0.1:8000/auth/obtain_token/admin/</a> - get a token</li>
+        <li><a href="http://127.0.0.1:8000/auth/users/">http://127.0.0.1:8000/auth/users/</a> - create a user</li>
```
[the remark this answers is not in the document]
Comment (sebbel) [reply cbv3_django_prototype/cbv3_django_prototype/templates/base.html]: That's a lovely idea, do you know where I could pull the values from?

File: cbv3_django_prototype/cbv3_django_prototype/templates/base.html (hunk 1 above)
Comment (sebbel) [reply cbv3_django_prototype/cbv3_django_prototype/templates/base.html]: that's a good catch, thanks @sunu  :)

File: cbv3_django_prototype/cbv3_django_prototype/templates/base.html [hunk 3]
```diff
@@ -83,7 +83,22 @@
       {% endif %}
 
       {% block content %}
-        <p>Use this document as a way to quick start any new project.</p>
+      <h1>Welcome to the codebuddies django-concept project</h1>
+      <p> Please find some useful commands below to help you get started on Django development</p>
+      <h2>Application urls</h2>
+      <ul>
+        <li><a href="http://127.0.0.1:8000/api/v1/resources/">http://127.0.0.1:8000/api/v1/resources/ </a> - browse the API for resources</li>
```
Comment (billglover) [inline cbv3_django_prototype/cbv3_django_prototype/templates/base.html]: An example would be:

```html
<a href="/api/v1/resources/">/api/v1/resources/</a>  - browse the API for resources
```

**Comment by sebbel** [comment]
@billglover @sunu changes implement, review at your convenience : )

**Pull request merged**